airflow/decorators/__init__.pyi

```diff
@@ -124,6 +126,40 @@ class TaskDecoratorCollection:
     @overload
     def virtualenv(self, python_callable: Callable[FParams, FReturn]) -> Task[FParams, FReturn]: ...
     @overload
+    def external_python(
+        self,
+        *,
+        python_fspath: str = None,
```

Comment:
I wonder if we should just call this argument `python`. Also this probably should not have `= None`.

Reply:
Yeah. I wanted to use `python` here but thought it was a bit `bare` but if it feels natural for you as well, I will change it.